packages/core/admin/admin/src/components/AutoReloadOverlayBlockerProvider/index.js

```diff
@@ -1,6 +1,6 @@
 import React, { useEffect, useRef, useState } from 'react';
 import PropTypes from 'prop-types';
-import { AutoReloadOverlayBockerContext } from '@strapi/helper-plugin';
+import { AutoReloadOverlayBlockerContext } from '@strapi/helper-plugin';
```

Comment:
I guess this one is a bit breaking change we cannot change the name of an exported component. Could you also maintain the old export name in the @strapi/helper-plugin/lib/index.js so plugin developers that are using it won't have breaking change?

Reply:
I think I've made the appropriate changes for this, but, I haven't done a lot of work w/ renamed imports/exports recently, so it's quite possible I've done it wrong.